src/app/clusters/closure-dimension-server/closure-dimension-cluster-logic.cpp

```diff
@@ -533,7 +533,18 @@ Status ClusterLogic::HandleSetTargetCommand(Optional<Percent100ths> position, Op
                 position.Value() = limitRange.min;
             }
         }
-
+        Percent100ths resolution;
+        VerifyOrReturnError(GetResolution(resolution) == CHIP_NO_ERROR, Status::Failure,
+                            ChipLogError(AppServer, "Unable to get Resolution value in SetTargetCommand"));
+        // Check if position.Value() is an integer multiple of resolution, else round to nearest valid value
+        if (position.Value() % resolution != 0)
+        {
+            Percent100ths roundedPosition =
+                static_cast<Percent100ths>(((position.Value() + resolution / 2) / resolution) * resolution);
```

Comment:
Are we guaranteed that all the limitRange boundaries fall on multiples of `resolution`?  If not (or just to be safe), should this logic not run before the limitRange checks?